### ISSUE TITLE
Stop the timer box from bouncing when you click start

### ISSUE THREAD

**Issue by huycans** [Alexandria/match-three-game#29]
Currently, when the timer runs out, the green timer box will start bouncing. Clicking the green timer box will reset it, but clicking on the start button doesn't.  I can work on a fix for this if you want.

**Comment by aokuonghae** [comment]
I'd like to give this a shot. Is the box supposed to remain still once the timer is done?

**Comment by huycans** [comment]
Thanks @Alexandria. @aokuonghae Another issue you could tackle is showing a replay prompt when the game ended. Right now it's not clear what the player should do when the game ended.

**Comment by aokuonghae** [comment]
Awesome. I can work on a prompt then.

**Comment by huycans** [comment]
PR is ready for review
